### DISCUSSION TITLE
[Question]Editor element enlarge control

### DISCUSSION THREAD

**Discussion by santoshgistto** [hm21/pro_image_editor#129] [Q&A] [answered]
Do we have option to change editor element control position   ? 

<img width="654" alt="Screenshot 2024-06-21 at 12 14 38 PM" src="https://github.com/hm21/pro_image_editor/assets/12577046/2051310b-dad8-4ef4-b062-57b6d18eec5e">

**Comment by hm21** [comment, marked as answer]
Currently, this is not possible, the only thing you can do now is to change the icon and colors of it.
